tembo-stacks/src/stacks/specs/analytics.yaml

```diff
@@ -0,0 +1,61 @@
+name: Parade Analytics
```

Comment:
```suggestion
name: Analytics
```

Will be easier if it's one word no space.

Reply:
Thought this was a display name for some reason; fixed it.